### PR TITLE
Fix tests in bootstrap-5 branch. Fixes #582

### LINKED ISSUE #582: Fix tests in bootstrap-5 branch
Currently 8 tests fail in the bootstrap-5 branch: 
```
Finished in 1.009611s, 236.7248 runs/s, 274.3631 assertions/s.
239 runs, 277 assertions, 8 failures, 0 errors, 0 skips
```

I'll try to fix these.

### PR DESCRIPTION
**Pull request by thimo**
Fixes the rendering tests in the bootstrap-5 branch after Rails rendering changes.

Also includes changes to remove Rubocop offences.

### CODE REVIEW

**Comment by thimo** [comment]
It looks like I "fixed" some tests for local testing, which may or may not be done with Rails 6.1, but that fail on Travis CI against 5.2 and 6.0. Will try to figure out if I can get local testing to be done against these Rails versions.

**Comment by thimo** [comment]
Tests are green again with Rails 5.2 and 6.0, plus I've added testing against 6.1. Also increased the Ruby versions Travis tests against one minor version.

**Comment by thimo** [comment]
I just noted that the main branch has some testing issues with Github Actions. After a quick glance I think these are related to Rails 6.1 and fixed in this PR. Let me know if you want me to prepare a PR to apply the same changes to the main branch as well.

File: lib/bootstrap_form/form_group_builder.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ module FormGroupBuilder
 
     private
 
-    def form_group_builder(method, options, html_options=nil)
+    def form_group_builder(method, options, html_options=nil, &block)
```
Comment (lcreid) [inline lib/bootstrap_form/form_group_builder.rb:9]: I think the change is fine. Again, just curious about the motivation?

File: test/bootstrap_rich_text_area_test.rb [hunk 2]
```diff
@@ -12,11 +12,14 @@ class BootstrapRichTextAreaTest < ActionView::TestCase
...
         <label class="form-label" for="user_life_story">Life story</label>
         <input type="hidden" name="user[life_story]" id="user_life_story_trix_input_user"/>
-        <trix-editor id="user_life_story" data-blob-url-template="http://test.host/rails/active_storage/blobs/:signed_id/:filename" data-direct-upload-url="http://test.host/rails/active_storage/direct_uploads" input="user_life_story_trix_input_user" class="trix-content form-control"/>
-        </trix-editor>
+        <trix-editor id="user_life_story" data-blob-url-template="#{data_blob_url_template}" data-direct-upload-url="http://test.host/rails/active_storage/direct_uploads" input="user_life_story_trix_input_user" class="trix-content form-control" />
       </div>
       HTML
       assert_equivalent_xml expected, form_with_builder.rich_text_area(:life_story)
     end
+
```
Comment (lcreid) [inline test/bootstrap_rich_text_area_test.rb:20]: Nice!

**Review by lcreid — APPROVED** [review]
Nice work. Thanks for doing all this. It's very much appreciated.

**Comment by lcreid** [comment]
> I just noted that the main branch has some testing issues with Github Actions. After a quick glance I think these are related to Rails 6.1 and fixed in this PR. Let me know if you want me to prepare a PR to apply the same changes to the main branch as well.

That would be awesome if you could prepare a PR to fix the Rails 6.1 tests in the `main` branch. (I hope to change the name of the main branch this weekend.)

**Pull request merged**